btk/measure.py

```diff
@@ -299,6 +302,34 @@ def __next__(self):
                     deblended_images[key_name].append(
                         measure_output[j][i].get("deblended_images", None)
                     )
+                if isinstance(blend_output["blend_list"], dict):
```

Comment:
the part below looks a little complicated, can you add a comment saying what you are doing? Thanks!